sensor_msgs/Image depth_image
sensor_msgs/Image rgb_image
float32[] bbox
float32[] extrinsics
float32[] intrinsics
---
float32 score
float32 width
float32 height
float32 depth
float32[] translation
float32[] rotation
---
int32 status
